src/app/components/pages/TeacherRegistration.tsx

```diff
@@ -21,10 +21,38 @@ import useRegistration from "../hooks/useRegistration";
 import { RegistrationSubmit } from "../elements/inputs/RegistrationSubmit";
 import ReCAPTCHA from "react-google-recaptcha";
 import { Recaptcha } from "../elements/inputs/Recaptcha";
+import { Link } from "react-router-dom";
 
 const metaDescriptionCS =
   "Sign up for a free account and get powerful GCSE and A Level Computer Science resources and questions. For classwork, homework, and revision.";
 
+export const AreYouATeacher = ({ confirmTeacher }: { confirmTeacher: (arg0: boolean) => void }) => {
```

Comment:
I also think that the `confirmTeacher` name could be improved. Based on a [convention](https://stackoverflow.com/a/60048346) that I've seen in other projects, this could be called `onTeacherConfirmed`, for instance.

That way when you use this later on, it reads nicely:
```typescript
<AreYouATeacher onTeacherConfirmed={() => setRegistrationStep(1)} />
```